NSS-Day-Cohort-42/rare-media-moguls
# Users Post List rendered with edit and delete buttons

#### Changes Made
1. Added file `UserPostList.js` & `Post.js` to `posts` directory.
1. Modified `ApplicationViews.js` to include path routed to `PostForm` view.

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-users-posts
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user clicks on `My Posts`, they should be redirected to a the users posts page 
> User can click the `...` button to reveal `edit` and `delete` buttons  which should allow the user to edit or delete the post on click
> User can Add a new post by clicking add new post button
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
